packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -121,11 +116,15 @@ export class OrgList implements vscode.Disposable {
   }
 
   public async updateOrgList() {
-    const authInfoObjects = await this.getAuthInfoObjects();
-    if (!authInfoObjects) {
+    const orgAuthorizations:
+      | OrgAuthorization[]
+      | null = await this.getOrgAuthorizations();
+    if (!orgAuthorizations) {
       return null;
     }
-    const authUsernameList = await this.filterAuthInfo(authInfoObjects);
+    const authUsernameList = await this.filterAuthInfo(
+      orgAuthorizations as OrgAuthorization[]
```

Comment:
The `as OrgAuthorization[]` isn't needed.

Reply:
Also, `updateOrgList()` doesn't define a return type.  How about adding it and changing line 118 to:
`public async updateOrgList(): Promise<string[]>`